doc/ref_model/chapters/chapter05.md

```diff
@@ -170,7 +169,7 @@ This section will detail NFVI SW profiles and associated configurations for the
 | .conf | Feature | Type  | Basic | Network Intensive | Compute Intensive |
 |------------------|----------------|----------------|----------------|----------------|----------------|
 | nfvi.net.cfg.001 | vNIC interface | IO virtualisation | virtio1.1 |  virtio1.1, i40evf (Intel driver for VF SR-IOV) |  virtio1.1, i40evf (Intel driver for VF SR-IOV) |
```

Comment:
Please remove i40evf (Intel driver for VF SR-IOV) as this has been removed from RM ch05

Reply:
Okay, basically you ask me to merge with master. I can do that.